internal/testgen/gen_contains.go

```diff
@@ -43,10 +43,26 @@ func (g ContainsTestsGenerator) TemplateData() any {
 			{Fn: "NotContains", Argsf: `s, "456"`},
 		},
 		IgnoredAssertions: []Assertion{
-			{Fn: "Contains", Argsf: `errSentinel.Error(), "user"`},
-			{Fn: "Equal", Argsf: `strings.Contains(s, "abc123"), true`},
-			{Fn: "False", Argsf: `!strings.Contains(s, "abc123")`},
-			{Fn: "True", Argsf: `!strings.Contains(s, "456")`},
+			{
+				Fn: "Contains",
+				Argsf: `errSentinel.Error(), "user"`,
+				ReportMsgf: "TODO: fix https://github.com/Antonboom/testifylint/issues/47",
+			},
+			{
+				Fn: "Equal",
+				Argsf: `strings.Contains(s, "abc123"), true`,
+				ReportMsgf: "Handled by bool-compare",
+			},
+			{
+				Fn: "False",
+				Argsf: `!strings.Contains(s, "abc123")`,
+				ReportMsgf: "Handled by bool-compare",
+			},
+			{
+				Fn: "True",
+				Argsf: `!strings.Contains(s, "456")`,
+				ReportMsgf: "Handled by bool-compare",
+			},
```

Comment:
I'm surprised to see gen_contains updated and pushed, but that the generated files weren't